website/docs/reference/widgets/chart.md

```diff
@@ -94,6 +94,7 @@ Let’s use a line chart to visualize the data.
 As you can see in the video, we use the `Chart Series` property head to supply the data and details related to identifying the data points.
 
 * `Series Title` - the name of the series. In the example above, it is _**Total Bugs**_.
+* `Series Color` - a color picker input to select the color of the series.
```

Comment:
@souma-ghosh please add that the series color property is not yet available for pie chart yet